### ISSUE TITLE
regression: podman login: root authfile path

### ISSUE THREAD

**Issue by edsantiago** [containers/podman#11725]
#11195 (my guess) broke `podman login`, in the sense that we can no longer share credentials with skopeo.
```console
# env|grep XDG_R
XDG_RUNTIME_DIR=/run/user/0

[podman 3.3]
# podman login quay.io
...
# find /run -name auth.json
/run/user/0/containers/auth.json       <---- this is good. It respects XDG

[podman 3.4.0-0.4.rc1.fc36]
# podman login quay.io
...
# find /run -name auth.json
/run/containers/0/auth.json            <---- this is BAD
```
This is caught by one of the system tests in `150-login.bats`. The test is passing in CI. My gut feel is that CI doesn't set the magic `XDG` envariables.

**Comment by vrothberg** [comment]
@Luap99 PTAL

**Comment by Luap99** [comment]
I guess we have to revert this commit.
I have another way to fix the original dnsname issue for podman 4.0 but 3.4 would then still have the problem with the dnsname plugin not properly cleanup left over files.

**Comment by Luap99** [comment]
PR #11733
